README.md

```diff
@@ -52,8 +52,14 @@ Chat Room
 
 Want to chat with other members of the PowerShell community?
 
+We have a Gitter Room which you can join below
 [![Join the chat at https://gitter.im/PowerShell/PowerShell](https://badges.gitter.im/PowerShell/PowerShell.svg)](https://gitter.im/PowerShell/PowerShell?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+There is also the community driven PowerShell Slack Team where there are over 1500 members already signed up with a number of channels for discussion on PowerShell topics including azure, dsc, ps-core (for discussionson topics mainly relating to this repository), ps-contributors and irc-bridge
```

Comment:
Fair point - I reviewed just in VSCode Markdown and it didn't look so bad

I'll get this reduced